packages/Switch/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@paprika/switch",
+  "version": "0.0.1",
```

Comment:
Oops, no big deal, but we've been launching components as `0.1.0` to start with.